subcommand/server-acl-init/command.go

```diff
@@ -491,6 +501,108 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if len(c.flagIngressGatewayNames) > 0 {
+		// Create a token for each ingress gateway name. Each gateway needs a
+		// separate token because users may need to attach different policies
+		// to each gateway token depending on what the services it represents
+		for _, name := range c.flagIngressGatewayNames {
```

Comment:
Since all the code here is within the iteration over `c.flagIngressGatewayNames` we don't need the
```go
if len(c.flagIngressGatewayNames) > 0
```